tests/e2e/cucumber/features/smoke/spaces/participantManagement.ocis.feature

```diff
@@ -65,7 +65,6 @@ Feature: spaces participant management
     And "Alice" creates a public link for the resource "parent" using the sidebar panel
     And "Alice" edits the public link named "Link" of resource "parent" changing role to "Can edit"
     And "Anonymous" opens the public link "Link"
-    And "Anonymous" closes the editor
```

Comment:
@JanAckermann  There is a similar step in `link.feature`. If this is required to be removed then please also remove it from the link.feature file as well. This will make the link tests pass.
https://github.com/owncloud/web/blob/992f52250492f6645db6daa2eb7564ff4af4b6c1/tests/e2e/cucumber/features/smoke/link.feature#L62